plugins/inventory/cobbler.py

```diff
@@ -296,8 +303,45 @@ def parse(self, inventory, loader, path, cache=True):
                 self.inventory.add_child(self.group, hostname)
 
             # Add host variables
+            ip_address = None
+            ipv6_address = None
+            for (iname, ivalue) in iteritems(interfaces):
```

Comment:
While `iteritems()` is a bit more efficient with Python 2.7, using `items()` is IMO better readable:
```suggestion
            for iname, ivalue in interfaces.items():
```